### ISSUE TITLE
Clarification regarding `sinkFile` and `sinkFileBS`

### ISSUE THREAD

**Issue by bitc** [snoyberg/conduit#483]
Hello,

The function [sinkFileBS](https://hackage.haskell.org/package/conduit-1.3.4.2/docs/Data-Conduit-Combinators.html#v:sinkFileBS) says that it is:

> [sinkFile](https://hackage.haskell.org/package/conduit-1.3.4.2/docs/Data-Conduit-Combinators.html#v:sinkFile) specialized to [ByteString](https://hackage.haskell.org/package/bytestring-0.10.10.0/docs/Data-ByteString.html#t:ByteString) to help with type inference.

But it has the exact same type signature and implementation as [sinkFile](https://hackage.haskell.org/package/conduit-1.3.4.2/docs/Data-Conduit-Combinators.html#v:sinkFile). I am guessing that at one point in history there was a difference.

But it looks like now they are the same, so can one of these functions be deprecated, or the documentation clarified?

Thank you

**Comment by snoyberg** [comment]
Good catch. I think updating the docs to say that `sinkFileBS` is now redundant with `sinkFile` would be a good change. PR would be welcome :)
